beetsplug/spotify.py

```diff
@@ -164,6 +164,12 @@ def _handle_response(self, request_type, url, params=None):
                 )
                 self._authenticate()
                 return self._handle_response(request_type, url, params=params)
+            elif response.status_code == 429:
+                seconds = response.headers['Retry-After']
```

Comment:
Spotify does not seem to guarantee that this header is present; I think we should gracefully handle that case


```suggestion
                seconds = response.headers.get('Retry-After', DEFAULT_WAITING_TIME)
```

where I don't really know a good value for `DEFAULT_WAITING_TIME`. One of the links from your issue uses 5 seconds.

Reply:
Set the `DEFAULT_WAITING_TIME` to 5 for now (based on the other issue).